client/header/activity-panel/index.js

```diff
@@ -221,14 +237,23 @@ export class ActivityPanel extends Component {
 
 	renderPanel() {
 		const { isPanelOpen, currentTab, isPanelSwitching } = this.state;
-
 		const tab = find( this.getTabs(), { name: currentTab } );
+
 		if ( ! tab ) {
 			return (
 				<div className="woocommerce-layout__activity-panel-wrapper" />
 			);
 		}
 
+		const clearPanel = () => {
+			this.clearPanel();
+		};
+
+		if ( currentTab === 'setup' ) {
+			window.location = getAdminLink( 'admin.php?page=wc-admin' );
```

Comment:
Go to the main page if the setup tab was chosen.  This will go away when we implement the nice-to-have of showing the task list in the panel, so I haven't refactored tabs to support going to links for now.

Reply:
Can we use `getHistory().push( ... )` to navigate here? If I click this on a wc-admin page, a full page refresh occurs.

Reply:
Have done this in https://github.com/woocommerce/woocommerce-admin/pull/4853/commits/0f243b2dd50e255879270e9e7039da156c08af84